src/app/withdraw/components/withdraw-client.tsx

```diff
@@ -102,13 +102,14 @@ const Withdraw = () => {
   const getMaxError = (maxWithdrawal: number) => {
     return `Withdrawal exceeds current cap of ${maxWithdrawal.toLocaleString(undefined, { maximumFractionDigits: 8 })} BTC`;
   };
+  const config = useAtomValue(bridgeConfigAtom);
   const amountValidationSchema = useMemo(() => {
     const btcBalance = Number(satsBalance) / 1e8;
     const fee = maxFee! / 1e8;
     return yup.object().shape({
       amount: yup
         .number()
-        .min(0)
+        .min(config.WITHDRAW_MIN_AMOUNT_SATS)
```

Comment:
Shouldn't this be divided by `1e8` as well?

Reply:
correct